index.html

```diff
@@ -2161,10 +2234,20 @@ <h3>Presentations</h3>
           </dd>
           <dt><var>proof</var></dt>
           <dd>
-The <a>verifiable presentation</a> MAY include a <code>proof</code>
-<a>property</a>. If present, the value SHOULD be used to express a securing
-mechanism, such as those listed in the
-Verifiable Credentials Specifications Directory [[?VC-SPECS]].
+            <p>
+                The <a>verifiable presentation</a> MAY include a <code>proof</code>
```

Comment:
I feel that proof remains a property of data integrity, and would prefer to see normative guidance related to data integrity done in the data integrity specification.

Reply:
@OR13 similar comment as above: I've just reused the current draft, and I do not have a strong opinion either way. Would it be possible to move this into a separate issue and not hold up this one on this?